src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+
+
+const todoList =[{'id': 1, 'tittle': 'Grocery shopping'}, {'id': 2, 'tittle':'pack boxes'}, {'id':3, 'tittle':'buy headphones'}];
```

Comment:
you don't have to use quotes for key value

`todoList =[{id: 1, tittle: 'Grocery shopping'}, {id: 2, tittle:'pack boxes'}, {id:3, tittle:'buy headphones'}];`

refer to [w3school objects](https://www.w3schools.com/js/js_objects.asp)